docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+services:
+  flask:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: runner-dev
+    ports:
+      - "8000:8000"
+      - "5055:5055"
+    depends_on:
+      - mongo
+      - redis
+    volumes:
+      - .:/usr/src/app
+    environment:
```

Comment:
Since we are specifying the env-file in the make command to build & run compose, having all the env variables here is a duplication. Since we have a custom .env location `app.config`, maybe it would be easier to reference the path as per https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/